src/main/java/seedu/address/logic/commands/TrainingCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.training.Training;
+
+/**
+ * Adds a training session of players specified by the index.
+ */
+public class TrainingCommand extends Command {
+
+    public static final String COMMAND_WORD = "training";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds training session of people identified by the index numbers used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1 5 7";
+
+    public static final String TRAINING_ADD_SUCCESS = "Training successfully added.";
+
+    private Date date;
+    private List<Index> indexList;
+
+    public TrainingCommand(List<Index> indexList) {
+        this.date = new Date();
+        this.indexList = indexList;
+    }
+
+    public TrainingCommand(Date date, List<Index> indexList) {
+        this.date = date;
+        this.indexList = indexList;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
This is a really long method 😯.
Can consider abstracting some parts out for readability after `v1.2`.